tests/modules/ultra/pipeline/main.nf

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ULTRA_PIPELINE } from '../../../../modules/ultra/pipeline/main.nf' addParams( options: [:] )
+
+workflow test_ultra_pipeline {
+    def wget1    = "wget -O " + workflow.workDir + "/genome.gtf "         + params.test_data['homo_sapiens']['genome']['genome_gtf']
+    def wget2    = "wget -O " + workflow.workDir + "/test_hifi.fastq.gz " + params.test_data['homo_sapiens']['pacbio']['hifi']
+    def sort_gtf = "sort -n -k 1,1 -k4,4 -k5,5 -o " + workflow.workDir + "/genome_sort.gtf " + workflow.workDir + "/genome.gtf"
+    def gunzip   = "gunzip " + workflow.workDir + "/test_hifi.fastq.gz"
```

Comment:
Yeah ... I needed to add this patch to make the test functional. 
In its actual state, uLTRA does not support gz files, so it downloads test datasets and gunzip it. 
Also the gtf need to be sorted before being used. 
I agree it's not elegant, but it's seems the only way to do it with actual state of uLTRA, which still very recent.

Reply:
@drpatelh I updated the code to use GUNZIP and GFFREAD to deal with the extraction and sorting. But the test failed because it try to run without waiting for gunzip to finish its process. I can't figure out why. Some help would be appreciated on this problem.

Reply:
Problem solved. The test is now clean 👍

Reply:
Checked and [solution](https://github.com/sguizard/modules/blob/2fb43ac48a25c8b990b0dcb2bd80bbd71cb47ea4/tests/modules/ultra/pipeline/main.nf#L15-L20) looks neat